sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -15,6 +14,7 @@
 from sunpy.net.jsoc import JSOCClient, JSOCResponse
 from sunpy.util.exceptions import SunpyUserWarning
 
+TEST_EMAIL = "nabil.freij@gmail.com"
```

Comment:
can you just put this in conftest so we don't have to change it everywhere if we want to change it to something less-Nabil-specific later?

Reply:
Sure.